chapter1/rotateMatrix.js

```diff
@@ -0,0 +1,44 @@
+// Given an image represented by an NxN matrix, where each pixel in the image is 4 bytes, write a method to rotate the image by 90 degrees.
+
+/*
+Input:
+[
+  [1,2,3],
+  [4,5,6],
+  [7,8,9]
+];
+Output: 
+[
+  [7,4,1],
+  [8,5,2],
+  [9,6,3]
+];
+
+*/
+rotateMatrix = (arr) => {
+    var rotateArr = [];
```

Comment:
`var` variable declarations are not the new coolness. Always either use `const` or `let`. `const` being read-only, and `let` not read-only. Both have block scope (it's scope is defined between the nearest open/close brackets `{ ... }`).